packages/now-next/src/utils.ts

```diff
@@ -13,6 +13,7 @@ import {
   Lambda,
   Route,
 } from '@now/build-utils';
+import { isSymbolicLink } from '@now/build-utils/dist/fs/download';
```

Comment:
Let's export this from the build-utils index file since we use it in a couple builders

Reply:
Updated